contracts/oracle_controller/IOracleController.cairo

```diff
@@ -44,7 +47,8 @@ namespace IOracleController:
     end
 
     func update_oracle_implementation_active_status(
-            oracle_implementation_address : felt, is_active : felt):
+        oracle_implementation_address : felt, is_active : felt
+    ):
```

Comment:
These are quite annoying - in the future when we update our linter version let's do one PR where we only lint so that then we can move on with our lives

Reply:
Agree, having linter in CI will help